hieradata/node/dimm.cp.lsst.org.yaml

```diff
@@ -0,0 +1,33 @@
+profile::nm::connections:
+  eno1:
+    content: |
+      [connection]
+      id=eno1
+      uuid=250c1a0b-95c3-4798-a4df-d864e60d3b63
+      type=ethernet
+      autoconnect=true
+      interface-name=eno1
+
+      [ethernet]
+
+      [ipv4]
+      method=auto
+
+      [ipv6]
+      method=disabled
+  enp2s0:
+    content: |
+      [connection]
+      id=enp2s0
+      uuid=de9904c8-9577-1a17-36b1-34b94132f06a
+      type=ethernet
+      autoconnect=true
+      interface-name=enp2s0
+
+      [ethernet]
+
+      [ipv4]
+      method=auto
```

Comment:
Isn't the secondary NIC use for a point to point connection? Maybe ask Brian which IP it needs and leave it static

Reply:
very true sir, lets use this opportunity to leave it static with the correct ip/net, thanks